### PR TITLE
[T14] Create API route for getting all tutor classes

### PR DESCRIPTION
**Pull request by absirgue**
The endpoint is setup with the appropriate SQL request + data cleaning.

I switched the DiscordUserName field to DiscordID so it will work as soon as @TunaAtasayar managed to change that name in the database as well.

Conducted proper testing but please let me know if you find a case in which it doesn't work properly or in my SQL JOIN does not join the right things together.

Thanks dear reviewers!

### CODE REVIEW

File: queries/classes.js [hunk 1]
```diff
@@ -1,37 +1,90 @@
...
+   * @param {*} sql The mssql instance connected to the database currently used by the API.
+   * @param {*} res The object to send result to a given query sender.
+   * @param {*} tutorDiscordId The tutor's discord id.
+   */
+  getTutorClasses: function (sql, res, tutorDiscordId) {
+    const request = new sql.Request();
+    request
+      .input("discordUsername", sql.NVarChar, tutorDiscordId)
+      .query(
+        "SELECT LEVELS.Name as Level, CLASSES.Date, COURSES.Subject, STUDENTS.FirstName, STUDENTS.LastName, CLASSES.ID FROM CLASSES INNER JOIN COURSES on CLASSES.CourseID = COURSES.ID INNER JOIN STUDENTS on STUDENTS.ID = COURSES.StudentID INNER JOIN LEVELS on COURSES.LevelID = LEVELS.ID WHERE COURSES.TutorID = (SELECT ID FROM TUTORS WHERE DiscordID=@discordUsername) AND (CLASSES.Status = 'Empty' or CLASSES.Status = 'Rescheduled')",
```
Comment (Ali-Alkhars) [inline queries/classes.js:66]: in the sql query, the CLASSES.status should be "Unknown" instead of "Empty"

File: queries/classes.js [hunk 2]
```diff
@@ -1,37 +1,90 @@
...
+                .status(400)
+                .json({ error: "There is not class with that ID." });
+            }
+          }
+        }
+      );
+  },
+
+  /**
+   * Sends all the classes of a specific tutor that happened less than 10 days go or that will happen in the future and that have either 'Unknown' (the classes has just been created)
```
Comment (Ali-Alkhars) [inline queries/classes.js:46]: The comment says either Unknown or ???

**Review by Ali-Alkhars — CHANGES_REQUESTED** [review]
Nicely done! I think you might have done some mistakes without realising, so please check them. Thanks.

File: queries/classes.js (hunk 1 above)
Comment (absirgue) [reply queries/classes.js:69]: We changed the status wording from 'Unknown' to 'Empty' but forgot to change the comment

File: queries/classes.js (hunk 2 above)
Comment (absirgue) [reply queries/classes.js]: Sorry, added that in

**Review by TunaAtasayar — APPROVED** [review]
when changin discordusernames to discordID you forgot to change the name of the variables. but since they both remained the same the code works perfect - im just saying this for the sake of "clean code"
apart from that well done!

File: queries/classes.js (hunk 2 above)
Comment (Ali-Alkhars) [reply queries/classes.js]: It should be "Empty or Rescheduled" not "postponed"

**Review by Ali-Alkhars — APPROVED** [review]
Great job! 

About the status "Unknown", I guess the database still hasn't been updated to use "Empty" so when I tried changing your code into "Unknown" I got a different answer which is why I thought it was a mistake.

The comment still has a minor issue though, it should say "rescheduled" instead of "postponed". I will approve the PR but please make sure that you edit the comment first.

Well done Monsieur.

**Pull request merged**